tests/control/HTTPTest.php

```diff
@@ -17,17 +24,19 @@ public function testAddCacheHeaders() {
 		HTTP::add_cache_headers($response);
 		$this->assertNotEmpty($response->getHeader('Cache-Control'));
 
-		// Ensure max-age is zero for development.
-		Config::inst()->update('Director', 'environment_type', 'dev');
+		// Ensure cache is disabled and max-age is ignored when disabled (e.g. when dev)
```

Comment:
You've removed the `max-age` assetion below, so this comment is no longer substantiated

Reply:
Yeah I mangled that comment sorry.